src/components/MainContainer.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { Layout } from 'antd';
-import { margin, dimMultipliers } from '../utils/constants';
```

Comment:
@lxu213 Any reason why you walked back putting the aspect ratio dimensions into the constacts file?

Reply:
I have no idea how that happened... which is disturbing, but alas, I've restored this with the latest commits.